internal/ctrlflow/ssa.go

```diff
@@ -0,0 +1,41 @@
+package ctrlflow
+
+import (
+	"go/constant"
+	"go/types"
+	"reflect"
+	"unsafe"
+
+	"golang.org/x/tools/go/ssa"
+)
+
+func setUnexportableField(objRaw interface{}, name string, valRaw interface{}) {
```

Comment:
document what this does and why we need it.

also, I assume we should name it `setUnexportedField`.